src/rpcclient.h

```diff
@@ -6,7 +6,10 @@
 #ifndef BITCOIN_RPCCLIENT_H
 #define BITCOIN_RPCCLIENT_H
 
-#include <univalue.h>
+#include <string>
+#include <vector>
+
+class UniValue;
```

Comment:
This is not required. `rpcclient.cpp` has a `#include <univalue.h>`. Or do i miss something?